docs/source/workflow.rst

```diff
@@ -0,0 +1,94 @@
+.. _ersemworkflow:
+
+##############
+ERSEM Workflow
+##############
+
+As a rule of thumb, ERSEM developments should be made in the `public ERSEM <https://github.com/pmlmodelling/ersem>`_
+with the exception of specific project developments.
+
+Issue tracking
+--------------
+
+All developments, either private or public, are required to create an 
+`issue/issues <https://github.com/pmlmodelling/ersem/issues>`_ describing the problem or
+development required.
+
+For larger project developments that require multiple issues, it is often preferable to create a
+`milestone <https://github.com/pmlmodelling/ersem/milestones>`_ which you can then assign multiple
+issues too.
+
+Private developments
+--------------------
+
+The following come under private developments:
+
+1. Private project developments - these developments require significant scientific changes to the model,
+   and thus, are made privately.
+
+Workflow for private developments
++++++++++++++++++++++++++++++++++
+
+The workflow for private developments is as follows:
+
+1. Create an issue or issues in the `public ERSEM <https://github.com/pmlmodelling/ersem>`_ 
+   repo based on the development required.
+2. Create a `branch <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository>`_
+   from the `private ERSEM <https://github.com/pmlmodelling/ersem-dev>`_ repo. The naming
+   convention of the branch is as follows `XX-[description]` where `XX` is the issue ID (this can be found
+   after creation of the issue) and `[description]` is a one or two work description to identify the branch
+   with the corresponding issue.
+3. Commit all changes outlined in the issue to this branch and push branch to the private ERSEM repo.
+4. Create `pull request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request>`_
+   to merge the branch you have made the changes on into the `master` branch. In the creation of the pull request you
+   will need to assign specific people to review the changes you have made before it can be merged into the master branch.
+5. Once the updates have been approved by the review/reviewers they will merge the development into the master branch.
+6. When the private master branch is updated you will need to talk to the ERSEM repo manager to merge those changes back
+   to the public repo.
+
+Public developments
+-------------------
+
+The following come under public developments:
```

Comment:
It took me a while to understand the meaning of this sentence. Once I got it, it was clear, but maybe something like that would be clearer.
"the following cases are considered public developments, and they will need to follow a different workflow (see below):"